examples/hello-world/hello-world-data/src/main.rs

```diff
@@ -61,27 +62,24 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
             .join("test-chain-config.json")
     });
 
-    let contract_bin = opts.contract_bin.unwrap_or_else(|| {
-        Path::new(&manifest_dir)
-            .join("..")
-            .join("contracts")
-            .join("greeting")
-            .join("out")
-            .join("debug")
-            .join("greeting.bin")
-    });
-
     let host = opts
         .host
         .unwrap_or_else(|| defaults::FUEL_NODE_ADDR.to_string());
     println!("Using Fuel node at {host}",);
 
-    let (wallet, contract_id) = get_contract_id_with_host(
-        chain_config.as_os_str().to_str().unwrap(),
-        contract_bin.as_os_str().to_str().unwrap(),
-        host,
-    )
-    .await?;
+    let wallet_path_str = chain_config.as_os_str().to_str().unwrap();
+    let mut wallet =
+        WalletUnlocked::load_keystore(wallet_path_str, defaults::WALLET_PASSWORD, None)
+            .unwrap();
+
+    let provider = Provider::connect(defaults::FUEL_NODE_ADDR).await.unwrap();
+
+    wallet.set_provider(provider.clone());
+
+    let contract_id: Bech32ContractId =
+        "fuel18hchrf7f4hnpkl84sqf8k0sk8gcauzeemzwgweea8dgr7eachv4s86r9t9"
```

Comment:
@deekerno Where is this magic string coming from?

Reply:
Deploying the contract and getting the generated contract ID from the node logs. I definitely don't think that these strings should stay long term, but I do think that finding an elegant way of spawning a contract deployment while simultaneously getting the contract ID may delay this a bit longer.

Reply:
It seems like we may be able to use something like the first example from the SDK: https://fuellabs.github.io/fuels-rs/v0.36.1/providers/short-lived.html